docs/scos/dev/front-end-development/202307.0/marketplace/ui-components-library/datasources/datasource-trigger/datasource-trigger-change.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Datasource Trigger Change
+description: This document provides details about the Datasource Trigger Change service in the Components Library.
+template: concept-topic-template
+related:
+  - title: Datasource Dependable
+    link: docs/scos/dev/front-end-development/page.version/marketplace/ui-components-library/datasources/datasource-dependable.html
+  - title: Datasource Http
+    link: docs/scos/dev/front-end-development/page.version/marketplace/ui-components-library/datasources/datasource-http.html
+  - title: Datasource Inline Table
+    link: docs/scos/dev/front-end-development/page.version/marketplace/ui-components-library/datasources/datasource-inline-table.html
+  - title: Datasource Inline
+    link: docs/scos/dev/front-end-development/page.version/marketplace/ui-components-library/datasources/datasource-inline.html
+---
+
+This document explains the Datasource Trigger Change service in the Components Library.
+
+## Overview
+
+Datasource Trigger Change is an Angular service that extracts the value from an event trigger element and checks whether it meets a certain criteria. If the value is valid, it emits an object containing the value.  
+
+Check out an example usage of the Datasource Trigger Change.
+
+Service configuration:
+
+| ATTRIBUTE | DESCRIPTION |
+| - | - |
+|type |  A datasource type. |
+|event |  An event type triggered by element. |
+|debounce |  Delays the emission of values the next datasource (default is `300ms`). |
```

Comment:
Can you clarify the description? Looks like something is missing

Reply:
What exactly? Description looks clear

Reply:
I don't get how `the next datasource` relates to the preceding part of the sentence

Reply:
`datasource` is an angular service and not exactly `data source`

Reply:
Still don't get it, but I'll try to guess: 
```suggestion
|debounce |  Delays the emission of values of the next datasource (default is `300ms`). |
```